sniffer.cpp

```diff
@@ -0,0 +1,199 @@
+#include <arpa/inet.h>
+#include <getopt.h>
+#include <netinet/in.h>
+#include <netinet/ip.h>
+#include <netinet/ip_icmp.h>
+#include <netinet/tcp.h>
+#include <netinet/udp.h>
+#include <pcap.h>
+#include <signal.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/socket.h>
+#include <unistd.h>
```

Comment:
Need to change to cpp headers?

Reply:
Yes please. Only these two: <stdio.h> <stdlib.h>